Small tweak for configure.in/configure to properly detect <termios.h>,
making the package build on GNU/kFreeBSD. Thanks to Petr Salinger for
the patch.  Closes Bug#520076 for octave3.1.

 -- Rafael Laboissiere <rafael@debian.org>  Thu, 12 Mar 2009 23:32:03 +0100

--- octave3.1-3.1.54.orig/configure.in
+++ octave3.1-3.1.54/configure.in
@@ -1523,7 +1523,7 @@
 AC_LANG_POP(C++)
 
 have_termios_h=no
-AC_CHECK_HEADERS($TERMIOS_H, have_termios_h=yes)
+AC_CHECK_HEADERS(termios.h, have_termios_h=yes)
 AC_CHECK_HEADERS(termio.h, have_termio_h=yes, have_termio_h=no)
 AC_CHECK_HEADERS(sgtty.h, have_sgtty_h=yes, have_sgtty_h=no)
 AC_CHECK_HEADERS(glob.h, have_glob_h=yes, have_glob_h=no)
